src/Migrations/M036MigrateCoversContentBlockToPostsListBlock.php

```diff
@@ -0,0 +1,360 @@
+                    [],
+                    [
+                        Utils\Functions::create_new_block(
+                            Utils\Constants::BLOCK_FEAT_IMAGE,
+                            ['isLink' => true]
+                        ),
+                        Utils\Functions::create_group_block(
+                            [
+                                Utils\Functions::create_group_block(
+                                    [
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'post_tag', 'separator' => ' ']
+                                        ),
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_TERMS,
+                                            ['term' => 'category', 'separator' => ' | ']
+                                        ),
+                                    ],
+                                    ['className' => 'posts-list-meta']
+                                ),
+                                Utils\Functions::create_new_block(
+                                    Utils\Constants::BLOCK_TITLE
+                                ),
+                                Utils\Functions::create_new_block(
+                                    Utils\Constants::BLOCK_EXCERPT
+                                ),
+                                Utils\Functions::create_group_block(
+                                    [
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_AUTHOR,
+                                            ['isLink' => true]
+                                        ),
+                                        Utils\Functions::create_new_block(
+                                            Utils\Constants::BLOCK_DATE
+                                        ),
+                                    ],
+                                    ['className' => 'posts-list-meta',]
+                                ),
+                            ],
+                            []
+                        ),
+                    ]
+                ),
+            ],
+            [
+                'lock' => [
+                    'move' => true,
+                    'remove' => true,
+                ],
+            ]
+        );
+    }
+
+    /**
+     * Create and get a new navigation link block.
+     *
+     * @return array - The new block.
+     */
+    private static function get_nav_links_block(): array
+    {
+        return Utils\Functions::create_new_block(
+            Utils\Constants::BLOCK_NAV_LINK,
+            [
+                'label' => 'See all stories',
+                'url' => get_permalink(get_option('page_for_posts')),
```

Comment:
What happens if the News & Stories page isn't set in the settings?

Reply:
Nice catch. In that case, the link will not be added.